src/components/Breadcrumb/Breadcrumb.module.css

```diff
@@ -0,0 +1,253 @@
+          }
+        }
+      }
+
+      &.breadcrumbItemButtonConnected {
+        gap: 4px;
+        background-color: var(--item-bg);
+        padding: var(--spacing-padding-xs, 4px) var(--spacing-padding-sm, 8px);
+        border-radius: var(--shape-border-radius-md, 4px);
+        cursor: pointer;
+        transition: var(--item-bg-transition);
+
+        &:hover {
+          background-color: var(--item-bg-hover);
+        }
+
+        &:active {
+          background-color: var(--item-bg-active);
+        }
+      }
+
+      & .breadcrumbItemLabelText {
+        max-width: 250px;
+      }
+    }
+  }
+
+  & .breadcrumbHidden {
+    visibility: hidden;
+    position: absolute;
+    top: 0;
+    left: 0;
+    white-space: nowrap;
+  }
+}
+
+.dropdownMenuContainer {
+  width: 300px;
+  max-height: 224px;
+  overflow: auto;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-gap-xs, 4px);
+  padding: var(--spacing-padding-xs, 4px);
+  background-color: var(--palette-common-white, #ffffff);
+  border-radius: var(--shape-border-radius-lg, 8px);
+  box-shadow: 0 2px 4px 0 #0000000D, 0 1px 6px -1px #0000000D, 0 1px 2px 0 #00000014;
+
+  &.dropdownCollapseMenuContainer {
+    width: 200px;
+  }
+
+  & .noItemsContainer {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: var(--spacing-padding-sm, 8px);
+    box-sizing: border-box;
+
+    & > div {
+      color: var(--palette-common-grey-400, #ACACAC);
+    }
+  }
+
+  & .dropdownMenuSearch {
+    display: flex;
+    justify-content: center;
+
+    & :global(.mia-platform-input-affix-wrapper) {
+      &:hover {
+        border: 1px solid var(--palette-action-primary-hover, #1890FF);
+      }
+
+      &:global(.mia-platform-input-affix-wrapper-focused) {
+        border: 1px solid var(--palette-action-primary-active, #005dc9);
+        box-shadow: 0px 0px 0px 2px #005DC940;
+      }
+    }
+  }
+
+  & .dropdownMenu {
+    & :global(.mia-platform-dropdown-menu) {
+      border-radius: 0;
+      box-shadow: none;
+      padding: 0;
+
+      & :global(.mia-platform-dropdown-menu-item) {
+        padding: var(--spacing-padding-sm, 8px) !important;
+        gap: var(--spacing-gap-sm, 8px);
+
+        &:hover {
+          background-color: var(--palette-action-alternate-secondary-hover, #00000014) !important;
+          cursor: pointer;
+        }
+
+        &:global(.mia-platform-dropdown-menu-item-selected) {
+          background-color: var(--palette-action-alternate-primary-active, #1890ff14) !important;
+
+          &:hover {
+            background-color: var(--palette-action-alternate-primary-active, #1890ff14) !important;
+          }
+        }
+
+        &.noInteraction {
+          cursor: default !important;
+          background-color: white !important;
+        }
+
+        & :global(.mia-platform-dropdown-menu-title-content) {
```

Comment:
The global is stripped at compile time, so the space is needed because the selector targets child elements with the class, and not an element having both classes